src/ngraph/builder/quantization_util.hpp

```diff
@@ -99,6 +99,59 @@ namespace ngraph
                     (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
                 return scale;
             }
+
+            static inline float get_quantize_scale(const std::shared_ptr<Node> min_input,
+                                                   const std::shared_ptr<Node> max_input,
+                                                   bool is_signed)
+            {
+                auto min_input_const_op = std::static_pointer_cast<ngraph::op::Constant>(min_input);
+                auto max_input_const_op = std::static_pointer_cast<ngraph::op::Constant>(max_input);
```

Comment:
would it be safer to dynamic cast and check for nullptr?

Reply:
Yes. We shouldn't even be assuming they are constant.